internal/gapicgen/generator/gapics.go

```diff
@@ -617,33 +616,6 @@ func ParseAPIShortnames(googleapisDir string, confs []*MicrogenConfig, manualEnt
 	return shortnames, nil
 }
 
-func (g *GapicGenerator) findModifiedDirs() ([]string, error) {
-	log.Println("finding modifiled directories")
-	files, err := git.FindModifiedAndUntrackedFiles(g.googleCloudDir)
-	if err != nil {
-		return nil, err
-	}
-	dirs := map[string]bool{}
-	for _, file := range files {
-		dir := filepath.Dir(filepath.Join(g.googleCloudDir, file))
-		dirs[dir] = true
-	}
-
-	// Add modified dirs from genproto. Sometimes only a request struct will be
-	// updated, in these cases we should still make modifications the
-	// corresponding gapic directories.
-	for _, pkg := range g.modifiedPkgs {
-		dir := filepath.Join(g.googleCloudDir, pkg)
-		dirs[dir] = true
-	}
-
-	var dirList []string
-	for dir := range dirs {
-		dirList = append(dirList, dir)
-	}
-	return dirList, nil
-}
-
```

Comment:
I believe this was orphaned code that should have been removed in a previous PR. Maybe @codyoss can link back to that PR just for the record...

Reply:
Correct this is follow up removal that should of happened in https://github.com/googleapis/google-cloud-go/pull/5687